notes.md

```diff
@@ -0,0 +1,124 @@
+## other notes.
+
+### Terms
+
+  - Price per share approx. `$54`
+  - Nature of securities `Equity`
+  - Closing Date `May 30 (Ideally)`
+  - Amount of securities offered `Approx 55,000`
+
+### Implied Terms
+  - Statement as to the number of days left in the campaign `probably 30 but TBD`
+  - Offering goal `$2.5M`
+
+### Non-terms
+
+* didn't come across as super relevant right now. Closing soon can appear on May 23 perhaps, one week before close.
+
+### Releasing of funds and delayed founder liquidity-
+
+  - both of these concerns are alleviated by the corporate structure.
+
+      1. Invested funds must be used for intended purposes or shareholders can sue us, thereby alleviating the risk of immediately releasing funds.
+      2. We have delayed liquidity in the sense that the company won't own a `premine`, so if we don't continue to perform valuable work for the network, people won't agree to pay our fee on block rewards and fork.
```

Comment:
From https://medium.com/iconominet/why-icos-fail-1f9530a6d135 and https://medium.com/iconominet/ico-2-0-what-is-the-ideal-ico-ee9d285a8939 there is a mention of escrow, controlled released of funds, and delayed liquidity (#8-10). 

Are we going to be more specific on the relationship between stock earning dividends and Exo rewards?  By block rewards you mean USD value after Exo is converted to USD? I'm a bit confused on this point.

Reply:
Also, for funds in escrow, how will we show that funds taken from it are used for a requirements/feature, is it through a governance DApp in which when proposals are accepted by vote then the funds are taken from escrow?

Reply:
I think if we issue Tokens as soon as ETH is deposited to the Crowdsale contract then there is no need for an Escrow. We should explain on the website whether the stock will be voting stock, ear dividends etc.  This is not a matter for the smart contracts.

Reply:
@tavakyan escrow and therefore the controlled release of funds is not needed for projects using a corporate structure. The corporate structure provides sufficient protection of the investment. Delayed liquidity is achieved because we are not pre-mining, but that has more to do with internal policies and marketing than distributing stock tokens.

CO2KN, INC's revenue model is to earn Exo by receiving a small split of every block reward on the C4Coin network. This strategy creates delayed liquidity in the sense that instead of the founders instantaneously having 10,000,000 Exo tokens to liquidate, we must earn them over time through perpetuation of the network. If the network decides the work CO2KN is doing isn't valuable, they will take the existing tech and fork w/o the fee. Therefore, we must do valuable work to earn profit.

Re: stock dividends- the company will earn Exo. The company could pay dividends in Exo, or if the stockholders so vote, the company can sell Exo to USD and pay USD dividends.